src/USER-MISC/pair_ilp_graphene_hbn.cpp

```diff
@@ -723,7 +738,6 @@ void PairILPGrapheneHBN::ILP_neigh()
 
     ILP_firstneigh[i] = neighptr;
     ILP_numneigh[i] = n;
-    if (n == 0) error->all(FLERR,"Could not build neighbor list to calculate normals, please check your configuration");
     if (n > 3) error->all(FLERR,"There are too many neighbors for some atoms, please check your configuration");
```

Comment:
shouldn't this be a call the error->one(), i.e. what is needed in a situation, where the error condition would **not** always happen on *all* MPI ranks.